docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+This document is provided for informational purposes only. It represents the current product offerings and practices from Amazon Web Services (AWS) as of the date of issue of this document, which are subject to change without notice. Customers are responsible for making their own independent assessment of the information in this document and any use of AWS products or services, each of which is provided “as is” without warranty of any kind, whether express or implied. This document does not create any warranties, representations, contractual commitments, conditions, or assurances from AWS, its affiliates, suppliers, or licensors. The responsibilities and liabilities of AWS to its customers are controlled by AWS agreements, and this document is not part of, nor does it modify, any agreement between AWS and its customers.
+
+© 2021 Amazon Web Services, Inc. or its affiliates. All Rights Reserved. This work is licensed under a Creative Commons Attribution 4.0 International License.
+
+This AWS Content is provided subject to the terms of the AWS Customer Agreement available at http://aws.amazon.com/agreement or other written agreement between the Customer and either Amazon Web Services, Inc. or Amazon Web Services EMEA SARL or both.
+
+## Points of Contact
+
+Author: `Author Name`  
+Approver: `Approver Name`  
+Last Date Approved:   
+
+## Executive Summary
+This playbook outlines the process to identify owners of code resources, determine how they were exposed, who may have accessed the exposed code, determine impact of exposure, required remediation actions, and determine root cause of code exposure.
+
+## Potential Indicators of Compromise
+- DLP Alerts
+- Known code publication or exposure
+- Suspicious CloudTrail logs
```

Comment:
I see DLP and code publication mentioned here, but not elsewhere in the document? 

Did you want to focus on S3, or include these in procedures as well for a more complete document?

Reply:
Added DLP and CloudTrail details.